docs/networks-bridged.md

```diff
@@ -0,0 +1,35 @@
+## Configuring network in 'Bridged' mode for public networking
+
+This will allow you to setup a BOSH director with the interface bridged over to the local network.  
+At the end of this, you should have a BOSH director exposed to your LAN. Services deployed to the director should also be exposed to the LAN.
+
+**It is intended this is to be used for inhouse development or training purposes and is not considered production ready**
+
+No Virtualbox changes should be needed.  
+The following variables should be set with the `internal_ip` set to a static IP on the LAN.
+
+```
+internal_ip: 192.168.43.252
+internal_gw: 192.168.43.3
+internal_cidr: 192.168.43.0/24
+outbound_network_name: NatNetwork
+network_device: en0
```

Comment:
Maybe add a note that this network_device name can be found using `VBoxManage list bridgedifs` . In my case on MacOS I had to use `network_device: "en0: Wi-Fi (Wireless)"`.